developer_docs/translations.md

```diff
@@ -0,0 +1,53 @@
+## Translations Guidelines
+
+
+*How to contribute with Translations for p5 web editor*
+ 
+**General rules of thumb for translations**
+
+In order to simplify the translations process the following rules of thumb were used:
+
+**Technical Part**
+
+* There is only one file to translate all the texts in any specific language, which is located under the directory, in respective locale [subdirectory](https://github.com/processing/p5.js-web-editor/tree/develop/translations/locales)
+* The new language code must be added to [i18n.js](https://github.com/processing/p5.js-web-editor/blob/develop/client/i18n.js#L7)
```

Comment:
Can you give the path to the file in the link: 
```suggestion
* The new language code must be added to [client/i18n.js](https://github.com/processing/p5.js-web-editor/blob/develop/client/i18n.js#L7)
```

Reply:
Cool, I agree